gitlens/gl-gitkraken-browser-extension.md

```diff
@@ -0,0 +1,58 @@
+---
+
+title: GitKraken Browser Extension
+description: Learn all about the GitKraken Browser Extension.
+taxonomy:
+    category: gitlens
+
+---
+
+The GitKraken Browser Extension bridges your GitHub or GitLab repositories to GitKraken Client and GitLens in Visual Studio Code. With this browser extension enabled, you get options inside github.com and gitlab.com to open repositories and commits in GitKraken Client or GitLens in Visual Studio Code.
+
+<img src='/wp-content/uploads/gl-be-open-repo.gif' class='img-bordered img-responsive center'>
+
+***
+## Requirements
+
+* [Google Chrome](https://www.google.com/chrome/) or [Mozilla Firefox](https://www.mozilla.org/en-US/firefox/new/)
+* [GitKraken Client](https://www.gitkraken.com/git-client/try-free) or [GitLens](https://www.gitkraken.com/gitlens/try-free) for Visual Studio Code
+
+***
+
+## How to install
+
+TODO
```

Comment:
Do we need this section completed before this merges? Is it published automatically when merged?

Reply:
waiting until the extension is published and then manually writing this section, so I have something to actually link to. Will complete before merging.